Functions.Templates/Templates/HttpGET(CRUD)-CSharp/metadata.json

```diff
@@ -1,12 +1,13 @@
 {
     "defaultFunctionName": "HttpGET-CRUD-CSharp",
-    "description": "$HttpGETCRUDC_description",
-    "name": "HttpGET(CRUD)",
+    "description": "$HttpGETCRUD_description",
+    "name": "HTTP Get",
```

Comment:
seems weird to me to do `Get`. Normally its `GET`, but given our style guideliness I could see `get` as well. `Get` seems off.

Reply:
@mattchenderson

Reply:
`GET` is correct.